src/docs/asciidoc/resources/migration_guide_to_4.5.adoc

```diff
@@ -12,3 +12,9 @@
 
 OperatorFabric is now using docker compose v2, so you have to upgrade this tool to continue using opfab.
 You can find more information in the docker documentation : https://docs.docker.com/compose/migrate/
+
+== Use of `date-fns` instead of `Moment.js` library
+
+OperatorFabric is now using `date-fns` library instead of `Moment.js`. The format used for defining a formatted date string
+is now different. So, if needed, you have to modify your fields : `settings.dateTimeformat` and `settings.dateFormat`.
```

Comment:
Specify that the modification is to be done in web-ui.json

Reply:
Done.